lib/platform/github/user.ts

```diff
@@ -19,14 +20,15 @@ export async function getUserDetails(
   }
   try {
     const userData = (
-      await githubApi.getJson<{ login: string; name: string }>(
+      await githubApi.getJson<{ id: number; login: string; name: string }>(
         endpoint + 'user',
         {
           token,
         }
       )
     ).body;
     userDetails = {
+      id: userData.id,
```

Comment:
Where/how is this used? I think we need to handle the app case too, where we can't call the `/user` endpoint.

Reply:
This is unnecessary, I've removed it